### DISCUSSION TITLE
Fuel quantity difference BLOCK FUEL / FOB

### DISCUSSION THREAD

**Discussion by hanomax24** [flybywiresim/aircraft#2845] [Q&A]
I see a difference between the BLOCK FUEL indication on MCDU's Perf/W&B Page and the FOB value on ECAM Fuel Page. 

For example:

Fuel selected by Weight & Balance Tab: 5995 kg
BLOCK FUEL indication on MCDU's PERF/W&B page: 5995 kg
FOB indication on ECAM's Fuel Page: 5920 kg

Is that normal?

**Comment by Inaxair** [comment]
Can you describe how you achieved that result? Like what route, what conditions, etc.

Thanks!
Inaxair from FBW

**Comment by felixharnstrom** [comment]
(ping @Inaxair )

I'm noticing the same thing. Here I spawned cold and dark with 0 fuel and used the PERF/W&B Page to load 10000 
kg of fuel. Note that the ECAM says 9920 kg FOB. According to #2545, there is 65.6 kg of unusable fuel and the ECAM only shows the usable fuel, Note that due to rounding, the FOB value in the ECAM might be more than 65.6 kgs 'behind'. However, logging the FUEL TOTAL QUANTITY simvar shows that the true quantity is BLOCK FUEL minus ~65.6 kg as expected (you might need to zoom in a a bit to see the log in the upper right corner). 

![insim_fuel_10000l_with_fob_log](https://user-images.githubusercontent.com/5500848/107875887-b12a9200-6ec2-11eb-94d6-713ca933a445.PNG)

However, I also noticed that the PERF/W&B and the ECAM are only kept in sync up until 19046 kgs (the max amount in the sim). Up until this point, the ECAM shows the amount of fuel as what's loaded in the MCDU (and in the sim) minus unusable fuel.

![insim_fuel_19046l_with_fob_log](https://user-images.githubusercontent.com/5500848/107876055-ce139500-6ec3-11eb-8ed6-fd329a442b11.PNG)

After this, the ECAM is capped out at 18960 kg (and the sim at 19046 kg) but the MCDU allows you to load up to 21273 kg of fuel. This extra fuel doesn't seem to do anything.

![insim_fuel_21273l_with_fob_log](https://user-images.githubusercontent.com/5500848/107883199-d1237b00-6eed-11eb-98df-3f77cc168c5f.PNG)

This is because the MCDU has a hard-coded fuel capacity of 21273: https://github.com/flybywiresim/a32nx/pull/1865#discussion_r543625007

Based on the discussions in #2286, it seems that the ECAM behavior is intended (for now at least), but I'm not so sure about the 21273 kg limit in the MCDU. It would be great if someone more knowledgeable would weigh in on this :)
